### PR TITLE
Translation l10n_it and l10n_fr

### PR DESCRIPTION
**Pull request by Rudolfet**
Hello, 

I have translated your mod into Italian and French.

### CODE REVIEW

File: FS22_HudColor/translations/l10n_fr.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" standalone="no" ?>
+<l10n>
+  <translationContributors>Rudolfet</translationContributors>
+  <elements>
+    <e k="mod_title" v="HudColor"/>
+    <e k="settings_hud_color_label" v="Couleur HUD"/>
+    <e k="settings_hud_color_label_needs_restart" v="(Il est nécessaire de redémarrer la sauvegarde !!)"/>
+    <e k="settings_hud_color_description" v="Changer la couleur du HUD pour un autre préréglage !!! Après le changement, vous devez redémarrer la sauvegarde !!!"/>
```
Comment (Peppie84) [inline FS22_HudColor/translations/l10n_fr.xml]: Please add `.` after `préréglage`

File: FS22_HudColor/translations/l10n_it.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" standalone="no" ?>
+<l10n>
+  <translationContributors>Rudolfet</translationContributors>
+  <elements>
+    <e k="mod_title" v="HudColor"/>
+    <e k="settings_hud_color_label" v="Colore HUD"/>
+    <e k="settings_hud_color_label_needs_restart" v="(È necessario riavviare il salvataggio!!)"/>
+    <e k="settings_hud_color_description" v="Cambiare il colore dell'HUD con un altro preset.!!! Dopo la modifica, è necessario riavviare il salvataggio!!!"/>
```
Comment (Peppie84) [inline FS22_HudColor/translations/l10n_it.xml]: Please add a whitespace after `preset.` and After `salvataggio`

**Comment by Peppie84** [comment]
Hi @Rudolfet , thanks for your PR. 

Can you please also update the `CHANGELOG.md` on the section `## [Unreleased]`. Something like:
```
- Added translation for IT and FR by [Rudolfet](https://github.com/Rudolfet)
```

File: FS22_HudColor/translations/l10n_fr.xml (hunk 1 above)
Comment (Rudolfet) [reply FS22_HudColor/translations/l10n_fr.xml]: the exclamation mark is correct?

Comment (Peppie84) [reply FS22_HudColor/translations/l10n_fr.xml]: yes, for now. I'll change it later. Thanks

**Pull request merged**